README.md

```diff
@@ -4,33 +4,112 @@
 
 ## Terms and Conditions
 
-LAN Protocol. LiFi Labs, Inc. © 2015. All rights reserved. Usage of this documentation is bound by the [LIFX Developer Terms](http://www.lifx.com/pages/developer-terms-of-use).
+LAN Protocol. LiFi Labs, Inc. © 2021. All rights reserved. Usage of this documentation is bound by the [LIFX Developer Terms](http://www.lifx.com/pages/developer-terms-of-use).
 
-### Extended multizone
+### Available capabilities
 
-Multizone products like the LIFX Z and the LIFX Beam have an "Extended Multizone"
-api when you have new enough firmware. The version of firmware that enables this
-feature is specified by the `min_ext_mz_firmware` and
-`min_ext_mz_firmware_components` fields for a multizone product in products.json
+* `color` = The light changes physical appearance when the Hue value is changed
+* `chain` = The light may be connected to physically separated hardware
+  (currently only the LIFX Tile)
+* `matrix` = The light supports a 2D matrix of LEDs (the Tile and Candle)
+* `relays` = The device has relays for controlling physical power to something
+  (the LIFX Switch)
+* `buttons` = The device has physical buttons to press (the LIFX Switch)
+* `infrared` = The light supports emitting infrared light
+* `multizone` = The light supports a 1D linear array of LEDs (the Z and Beam)
+* `temperature_range` = An array of the minimum and maximum kelvin values this
+  device supports. If the numbers are the same then the device does not support
+  variable kelvin values. It is `null` for devices that don't have LEDs (the
+  LIFX Switch)
+* `extended_multizone` = The more capable `extended` API for multizone control
+
+### Determining capabilities
+
+To determine the capabilities of a device you need four values from the device:
+
+* `vendor_id`, this will likely be `1` which says it's a LIFX device
+* `product_id`, this has a different number per product
+* `firmware_major`, the major revision number for the firmware
+* `firmware_minor`, the minor revision number for the firmware
```

Comment:
I wonder if we should add timestamp as well since in reality a feature may be added in a particular RC and the only way to determine that is with the FW timestamp.

Reply:
mainly because in the public registry, we shouldn't have to care about release candidates!!!

Reply:
I agree, but the fact is we have release candidates in the field because the factory needs to start producing product before a release is finalized.  I agree it is not ideal, but that is the reality we live with.

Reply:
you do realise these comments are public right?